public/i18n/en.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
this  should be in simulator folder

Reply:
yes , desired the same but fetch api makes requests such as ```localhost/i18n/en.json```  in that case had to use public directory .

will modify this also as i change loading mechanism